src/drive/targets/mobile/README.md

```diff
@@ -27,9 +27,25 @@
 - Define your JAVA_HOME
   - Add `export JAVA_HOME=$(/usr/libexec/java_home)` in your `.zshrc` or `.bashrc` file
 - Add Android build-tools in your Path
-  - Add `export PATH="$ANDROID_HOME/build-tools:$PATH"` in your `.zshrc` or `.bashrc` file
+  - Add `export PATH="$ANDROID_HOME/build-tools/$(ls $ANDROID_HOME/build-tools | sort -V | tail -n 1):$PATH"` in your `.zshrc` or `.bashrc` file
+  - Adding `export PATH="$ANDROID_HOME/platform-tools:$PATH"` can be useful too to get tools such as `adb`, but not necessary
 
 
+#### Android > 9
+
+To build for android version > 9 you need to add a specific config line in `config.xml`
+
+- Inside `<edit-config file="app/src/main/AndroidManifest.xml" [...]></edit-config>`, add `<application android:usesCleartextTraffic="true" />`
+
+#### Cordova-android 9
+
+To build with cordova-android 9 plugin you have to use :
+
+- Node v14
+- Cordova 8.1.2
+- Cordova-android 9.1.0
+- Android SDK build-tools 30.0.3
+
```

Comment:
this should be at the top of the file here https://github.com/cozy/cozy-drive/blob/d0c89ef844bd4e26eef5f9ce1f2af7185ac2b53b/src/drive/targets/mobile/README.md#wrench-requirements

Reply:
Also, Cordova-android 9.1.0 is not a requirement since it'll be installed by cozy-drive. 

Node 14
Cordova 8 
Android SDK Build-tools 

are the requirements